src/libraries/Common/src/Interop/Windows/Gdi32/Interop.BitBlt.cs

```diff
@@ -8,8 +8,8 @@ internal static partial class Interop
 {
     internal static partial class Gdi32
     {
-        [DllImport(Libraries.Gdi32, ExactSpelling = true, SetLastError = true)]
-        public static extern int BitBlt(IntPtr hdc, int x, int y, int cx, int cy,
+        [GeneratedDllImport(Libraries.Gdi32, ExactSpelling = true, SetLastError = true)]
```

Comment:
I think we should get rid of all the `ExactSpelling = true` instance where it isn't needed.